memote/suite/results/result.py

```diff
@@ -38,3 +41,13 @@ def __init__(self, *args, **kwargs):
         super(MemoteResult, self).__init__(*args, **kwargs)
         self.meta = self.setdefault("meta", dict())
         self.cases = self.setdefault("tests", dict())
+        self.add_environment_information(self.meta)
```

Comment:
I remember now why I didn't have this in the `__init__` method, when we are loading an existing result from file or database, we should not overwrite the environment information.

Reply:
Gotcha! I've reorganised it so that the staticmethod is called explicitly only the when collecting results in the ResultCollectionPlugin which is instantiated only in `test_model`